dcpy/builds/load.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+from dcpy.utils import postgres
+from dcpy.connectors.edm import recipes
+
+
+def load_source_data(
+    pg_client: postgres.PostgresClient,
+    recipe_path: Path,
+    recipe_lock_path: Path,
```

Comment:
I think we probably always want to put the lockfile next to the recipe.yml. Maybe ✂️?

Reply:
done in [ca46774](https://github.com/NYCPlanning/data-engineering/pull/269/commits/ca46774cfbba5b04c35d0d75456231a564d16e4d)

could go further and construct the lockfile path in `recipes.plan` if prefered, but that'd force changes to `pluto_build.yml`

Reply:
yeah, I'm all for it. And pluto should be changed to use this, so all for it.

Reply:
mmmm looks like that'd be a more substantial change to `edm.recipes` than I'd like to do in this PR. since every recipes command after plan expects the lockfile path, they'd all have to be changed. it's a good idea but seems out of scope here

and since the pluto build action [does things](https://github.com/NYCPlanning/data-engineering/blob/main/.github/workflows/pluto_build.yml#L75-L95) in between `edm.recipes plan` and `edm.recipes import`, it'd take some pluto refactoring to use `builds.load.load_source_data`